ui/app/rif/pages/domainsPage/domainsPage.js

```diff
@@ -67,17 +67,17 @@ class DomainsScreen extends Component {
           <path d="M1 1L9 8.5L1 16" stroke="#5B2A92" strokeWidth="2"/>
         </svg>
       </span>
-    </div>
+    </li>
   }
 
   render () {
     if (this.props.domains && this.props.domains.length > 0) {
       return (
-        <div className={'domains-list'}>
+        <ul className={'domains-list'}>
```

Comment:
You can use just literal declaration instead.

Reply:
There i changed it